src/components/Text.js

```diff
@@ -1,18 +1,17 @@
-import React, { Component } from 'react';
+import React from 'react';
+import '../css/text.css';
 
-export class Text extends Component {
-	render() {
+function Text({textTitle, textPara, button}){
```

Comment:
Thoughts on `title, body, and buttonCopy`?

Reply:
just practicing destructuring